modules/devkit-support/src/test/java/org/mule/security/oauth/processor/OAuth2AuthorizeMessageProcessorTestCase.java

```diff
@@ -215,6 +222,9 @@ public String answer(InvocationOnMock invocation) throws Throwable
         private String customField;
         private String anotherCustomField;
 
+        @JsonProperty(annotatedPropertyName)
```

Comment:
is devkit supposed to generate such a field on the MP if the connector is annotated somehow?